packages/cli/package.json

```diff
@@ -97,8 +97,8 @@
   },
   "scripts": {
     "dev": "rm -rf .mailing && yarn prisma migrate dev && src/dev.js",
-    "ci:server": "rm -rf .mailing && yarn prisma generate && yarn prisma migrate deploy && NODE_ENV=test src/dev.js > test.log",
-    "ci:server:nohup": "rm -rf .mailing && NODE_ENV=test nohup src/dev.js --quiet > /dev/null 2>&1 &"
+    "ci:server": "rm -rf .mailing && yarn prisma generate && yarn prisma migrate deploy && NEXT_PUBLIC_MAILING_SKIP_AUTH=false NODE_ENV=test src/dev.js > test.log",
+    "ci:server:nohup": "rm -rf .mailing && NEXT_PUBLIC_MAILING_SKIP_AUTH=false NODE_ENV=test nohup src/dev.js --quiet > /dev/null 2>&1 &"
```

Comment:
this makes it so that in integration tests, api key is required

Reply:
Does `yarn test:integration` work locally under this scheme?  You may also have to set this variable in `integration:servers:start`

Reply:
yep, will update

Reply:
ah nvm, that just calls this